kolibri/core/assets/src/core-app/mediator.js

```diff
@@ -82,13 +93,16 @@ module.exports = class Mediator {
     this._executeCallbackBuffer(kolibriModule);
     logging.info(`KolibriModule: ${kolibriModule.name} registered`);
     this.emit('kolibri_register', kolibriModule);
-    if (this._ready) {
-      kolibriModule.ready();
-    } else {
-      this._eventDispatcher.$once('ready', () => {
+    const ready = () => {
```

Comment:
Yes, so we make sure that any relevant language assets have been fetched before we declare the module ready!

That ensures that we don't have modules running before they have their translations available (as I have not got 'reactive' translation working).